src/controllers/article.js

```diff
@@ -22,5 +23,53 @@ export default {
     const articleData = await Articles.createComment({ articleId, comment });
 
     return response.status(201).json({ status: 'success', data: articleData, message: 'Comment added succesfully' });
+  },
+
+  /**
+      * controller for adding article to bookmark
```

Comment:
Please align this properly

Reply:
Thanks for the review. I have implemented all the feedback.